Examples/IntelliTect.TestTools.Selenate.Examples/Pages/ShadowDomPage.cs

```diff
@@ -0,0 +1,22 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace IntelliTect.TestTools.Selenate.Examples.Pages
```

Comment:
Given that the project was updated to C# 10, do you also want to leverage 10 features like file scoped namespaces?

Reply:
Probably. I'm doing my best to keep the example project as "best practice" as possible.

Reply:
Converted to file scoped namespaces, implemented a few global usings (however I did it in a Global.cs file, not in the csproj. Not sure if there's a best practice here @Keboo ?)

Reply:
Left a comment on that file. There is not agreed upon best practice, but I have a minority opinion that says they should be managed in MSBuild files.

Reply:
I'll change 'em over with the next update. Thank you!